cognite/client/_api/labels.py

```diff
@@ -156,3 +156,37 @@ def delete(self, external_id: str | SequenceNotStr[str] | None = None) -> None:
                 >>> client.labels.delete(external_id=["big_pump", "small_pump"])
         """
         self._delete_multiple(identifiers=IdentifierSequence.load(external_ids=external_id), wrap_ids=True)
+
+    @overload
+    def retrieve(self, external_id: str) -> LabelDefinition: ...
```

Comment:
This should still be `LabelDefinition | None`.

Reply:
@erlendvollset reverted now